hudi-common/src/main/avro/HoodieArchivedMetaEntry.avsc

```diff
@@ -37,6 +37,7 @@
          "default": null
       },
       {
+         /** DEPRECATED **/
```

Comment:
Yeah, This is was intended to be equivalent of HoodieCommitMetadata but since we are using HoodieCommitMetadata for storing results of compaction, this is never used.